frontend/src/pages/Teachers.vue

```diff
@@ -137,6 +144,24 @@ export default {
         };
       });
     },
+
+    exportTeachersData() {
+      return this.teachers.map((teacher) => {
+        var language2 = teacher.secondLanguage
+          ? teacher.secondLanguage.NativeLanguage
+          : "";
+        return {
+          TeacherID: `${teacher.TeacherID}`,
+          FullName: `${teacher.FullName}`,
+          CreatedAt: `${teacher.created_at}`,
+          Status: `${teacher.status.Description}`,
+          PhoneNumber: `${teacher.PhoneNumber}`,
+          Email: `${teacher.Email}`,
+          Language1: `${teacher.nativeLanguage.NativeLanguage}`,
```

Comment:
Not a big deal but I don't think you need the string templates here.

```js
{
  TeacherID: teacher.TeacherID,
  FullName: teacher.FullName,
  // ...
}
```

Reply:
Have removed the string templates. Thank you Thomas!